package.json

```diff
@@ -83,7 +83,9 @@
     "test:utilities": "(cd packages/airnode-utilities && yarn run test)",
     "test:validator": "(cd packages/airnode-validator && yarn run test)"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
I think this is a mistake?

Reply:
This is interesting as I hadn't touched the file; it might've been the prettifier. We should definitely revert that.